src/WebJobs.Script/Description/FunctionDescriptorProvider.cs

```diff
@@ -110,17 +110,7 @@ public virtual bool TryCreate(FunctionMetadata functionMetadata, out FunctionDes
         protected virtual ParameterDescriptor CreateTriggerParameter(BindingMetadata triggerMetadata, Type parameterType = null)
         {
             ParameterDescriptor triggerParameter = null;
-            string type = triggerMetadata.Type.ToLowerInvariant();
-            switch (type)
-            {
-                case "manualtrigger":
```

Comment:
That was the last of our trigger bindings that weren't actual core SDK bindings/extensions :)